package-res/resources/web/pentaho/type/filter/_doc/_namespace.jsdoc

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This namespace contains types for expressing
+ * [intensional]{@link https://en.wikipedia.org/wiki/Intensional_definition}.
+ * filters of complex values or lists of.
```

Comment:
The phrasing "lists of" sounds a bit awkward to me. Did @jheck123 take a look at this?